src/components/AdminPanel/AdminPanel.js

```diff
@@ -0,0 +1,83 @@
+import React from "react";
+import "./AdminPanel.scss";
+import { Container, Alert, Card, Col, Row, Button } from "react-bootstrap";
+import img_oil from "../../assets/images/ola_ola_palm_oil.jpg";
+import img_logo from "../../assets/images/logo2.png";
+import { Link } from "react-router-dom";
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogContentText from '@material-ui/core/DialogContentText';
+import DialogTitle from '@material-ui/core/DialogTitle';
+
+//////////////////////////////////////////////////////////////////////
+class AdminPanel extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      open:false
+    };
+  }
+//////////////////////////////////////////////////////////////////////
+  handleOnClickItems=(item, userRole)=>{
+    if (item =="suggest" && userRole=="admin"){
+      return (window.location.href = "/ViewSuggestedMeals");
+    }else{
+      this.setState({open:true});
+    }
+  }
+
+  handleClose = () => { this.setState({open: false});};
+    //////////////////////////////////////////////////////////////////////
+  render() {
+    const userRole = window.localStorage.getItem("userRole");
+    console.log("yyy:", userRole);
+    return (
+      <Container className="admin-page">
+          <Row>
+            <Col md={12} className="admin-title-region">
+                  <div className="admin-title"><div style={{width: "100%"}}>Admin Dashboard</div></div>
+            </Col>
+
+            <Col md={4} className="admin-item-panel">
+                <div className="item-card">
+                  <div className="admin-item-title"><div style={{width: "100%"}}>INVENTORY</div></div>
```

Comment:
What is the difference between AdminPanel and AdminManagement ?

Reply:
As you see the code, you can see that AdminManagement is empty. Thus you can remove it

Reply:
Why is there no AdminPanel to the server repository ?

Reply:
<img width="1440" alt="adminManagement same as AdminPanel ?" src="https://user-images.githubusercontent.com/6497687/86710546-f18e6d00-bfcf-11ea-8f4c-92c5aa136884.png">

The AdminManagement file is not empty, I see it in this client folder here. If it is not needed,  can you remove it from the git commit so I can only pull the file that is required.

Reply:
As you see AdminManagement.scss, it is empty. That is, AdminManagement page was not used in this code at all.